org.civicrm.reqangular/js/build.js

```diff
@@ -30,7 +30,8 @@
     'common/lodash': 'common/vendor/lodash.min',
     'common/moment': 'common/vendor/moment.min',
     'common/mocks': '../test/mocks',
-    'common/vendor/perfect-scrollbar': 'common/vendor/perfect-scrollbar.min'
+    'common/vendor/perfect-scrollbar': 'common/vendor/perfect-scrollbar.min',
+    'common/angular-spectrum-colorpicker': '../../node_modules/angular-spectrum-colorpicker/dist/angular-spectrum-colorpicker.min'
```

Comment:
Good question. 2KB, don't think it will affect the performance.

![image](https://user-images.githubusercontent.com/3973243/47031218-586a5380-d167-11e8-84c5-51ef7bc06200.png)

The approach was confirmed with @AkA84.